ecs.php

```diff
@@ -10,5 +11,9 @@
         __DIR__ . '/ecs.php',
     ]);
 
+    $ecsConfig->skip([
+        InlineDocCommentDeclarationSniff::class . '.MissingVariable',
+    ]);
```

Comment:
What's reasoning behind this?

Reply:
I had an issue with ecs, but I solved it. Removing